src/docs/admin/references/NodeSourceInfrastructureReference.adoc

```diff
@@ -480,12 +491,23 @@ Other properties for the node deployment in the *Create OpenStack Node Source* a
     	    	
     -   *additionalProperties:* Additional Java command properties to be added when starting each ProActive node JVM in OpenStack instances (e.g. \"-Dpropertyname=propertyvalue\").
     
+    -   *nodesInitDelay:* Estimated startup time (expressed in millisecond) of the nodes, including the startup time of VMs. After this timeout expired, the node will be considered as lost.
+
+    -   *startupScript:* VM startup script to launch the ProActive nodes. 
+        Usually, this script is expected to download and install the Java Runtime Environment, then download and launch ProActive node.jar, if not already provided by the VM image.
+        The script can use <<_cloud_infrastructure_startup_script,some arguments>>. For example, it can use `%nodeJarUrl%` to represent the full URL path for downloading the _node.jar_ file. These arguments will be interpreted by Openstack Infrastructure later.
```

Comment:
Suggestion: This script should launch ProActive node.jar. It shis also expected to download and install the Java Runtime Environment, and download and launch ProActive node.jar, if they are not already provided by the VM image.